app/src/main/java/org/imozerov/streetartview/network/FetchService.kt

```diff
@@ -30,15 +29,20 @@ class FetchService : IntentService("FetchService") {
     }
 
     private fun handleStartFetchAction() {
-        val response = restClient.artWorksEndpoint.list().execute()
-        if (!response.isSuccess) {
-            Log.e(TAG, "Unable to fetch data from server! ${response.code()}")
-            return
-        }
+        try {
+            val response = restClient.artWorksEndpoint.list().execute()
+
+            if (!response.isSuccess) {
+                Log.e(TAG, "Unable to fetch data from server! ${response.code()}")
+                return
+            }
 
-        val responseJson = response.body().artworks
+            val responseJson = response.body().artworks
 
-        dataSource.insert(responseJson)
+            dataSource.insert(responseJson)
+        } catch (exception: java.net.UnknownHostException) {
+            Log.e(TAG, "Unknown host exception. It can be caused by turned off network connection")
```

Comment:
Please log with 
`Log.w(TAG, "Unable to sync art objects with server", exception)`